packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -129,8 +129,24 @@ const { t } = injectI18n();
       </div>
     </label>
 
-    <div v-if="props.message || shouldShowCounter" class="onyx-input__footer onyx-text--small">
-      <span v-if="props.message" class="onyx-truncation-ellipsis">{{ props.message }}</span>
+    <div
+      v-if="props.message || errorMessages.shortMessage || shouldShowCounter"
+      class="onyx-input__footer onyx-text--small"
+    >
+      <span v-if="errorMessages.shortMessage" class="onyx-input__error-message">
+        <span class="onyx-truncation-ellipsis">{{ errorMessages.shortMessage }}</span>
+
+        <OnyxInfoTooltip
+          v-if="errorMessages.longMessage"
+          class="onyx-input__message-tooltip"
+          color="danger"
+          style="color: red"
```

Comment:
Is this a leftover?

Reply:
which part are you referring to? the extra v-if is needed, you could have only the short error message

Reply:
aye 🤦🏻‍♀️
```suggestion
```